<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppsView</class>
 <widget class="QDialog" name="AppsView">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>323</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>323</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Apps View</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lApps">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Applications:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>35</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Id, Name, Category, License</string>
   </property>
  </widget>
  <widget class="QLabel" name="lFilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filter: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="orderAlph">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>5</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A-Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="orderCantPkg">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>5</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string># Pkg</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lCount">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(0, 0, 0, 150);
border: 5px; border-radius:20px;
color:white;
font: bold 10pt &quot;MS Shell Dlg 2&quot;;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <action name="actionViewPackages">
   <property name="text">
    <string>Show Packages</string>
   </property>
  </action>
  <action name="actionDeleteApp">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
